db/sql/src/registry.rs

```diff
@@ -16,7 +17,10 @@ pub trait HoprDbRegistryOperations {
         -> Result<()>;
 
     /// Returns `true` if the given node is allowed in network registry.
-    async fn is_allowed_in_network_registry<'a>(&'a self, tx: OptTx<'a>, address: Address) -> Result<bool>;
+    async fn is_allowed_in_network_registry<'a, T>(&'a self, tx: OptTx<'a>, address_like: &T) -> Result<bool>
+    where
+        Address: TryFrom<T>,
```

Comment:
On template bounds, `TryInto` shall be preferred:

`T: TryInto<Address> + Clone + Sync`

Reply:
I can fix this on `master`. I don't want to mess up the proper port.